### DISCUSSION TITLE
OpenAI o1-mini max_tokens

### DISCUSSION THREAD

**Discussion by eniompw** [community/community#144535] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I get the following error when trying to use OpenAI o1-mini:

Unsupported parameter: 'max_tokens' is not supported with this model. Use 'max_completion_tokens' instead.

**Comment by bytexenon** [comment]
+1,
doesn't work for me too

**Comment by mcasalaina** [comment]
The team is looking into this issue now

**Reply by eniompw** [reply, marked as answer]
seems fixed now thx.
